child_compassion/demo/compassion.child.csv

```diff
@@ -0,0 +1,67 @@
+id,code,name
+child_01,"BR2150576","Marilyn Monroe"
+child_02,"RW1440285","Abraham Lincoln"
```

Comment:
Maybe I keep just one case where a country has two projects (it will be one more test)